hyperspy/misc/hyperspy_magics.py

```diff
@@ -26,10 +37,9 @@ def hyperspy(self, line):
 
         Positional arguments:
         ---------------------
-            toolkit
+            toolkit : {qt4, gtk, wx, tk, None}
```

Comment:
> Also, the only recognised flat is -r - if anything else is passed, it just stops at toolkit check (where we check if the passed toolkit is supported).

Yes, but the error is not very friendly for users. For comparison this is how the matplotlib magic fails:

```python
In [1]: %matplotlib -a
UsageError: unrecognized arguments: -a

```

> Finally, can we really say that we allow gtk and tk? Or just "it won't crash, but won't work either"? If they don't give full functionality, I see no point allowing them

I think that it is good to keep them because HyperSpy is useful without the GUI elements and someone may want to use it e.g. with the GTK backend for some reason...